app/models/journeyDomain/identification/IdentificationDomain.scala

```diff
@@ -33,7 +33,7 @@ case class IdentificationDomain(
 ) extends JourneyDomainModel {
 
   override def routeIfCompleted(userAnswers: UserAnswers, mode: Mode, stage: Stage): Option[Call] =
-    Some(controllers.identification.routes.CheckIdentificationAnswersController.onPageLoad(userAnswers.mrn))
+    Some(controllers.locationOfGoods.routes.TypeOfLocationController.onPageLoad(userAnswers.mrn, mode))
```

Comment:
This would work for NormalMode, but in CheckMode we'd expect it to go back to the final CYA for the entire arrival. We could possibly use `Some(UserAnswersNavigator.nextPage[ArrivalDomain](userAnswers, mode))` here instead.